packages/bundle-analyzer/src/assets.ts

```diff
@@ -6,13 +6,14 @@ import {
   type Asset,
 } from "@codecov/bundler-plugin-core";
 import { fileURLToPath } from "node:url";
+import micromatch from "micromatch";
 
-// Resolve __dirname and __filename for both CommonJS and ESModules
 let fileName: string;
 let __dirname: string;
 const isCommonJS =
   typeof module !== "undefined" && module.exports !== undefined;
 if (isCommonJS) {
+  // CommonJS environment
```

Comment:
IMO this comment is redundant because we have `if (isCommonJS) {` right above it.

Maybe we rename the variable is `isCommonJSEnvironment`?

Reply:
yeah that's true, updated